crates/hyperqueue/src/worker/start.rs

```diff
@@ -504,8 +504,13 @@ fn try_get_pbs_info() -> anyhow::Result<ManagerInfo> {
     let manager_job_id =
         std::env::var("PBS_JOBID").expect("PBS_JOBID not found in environment variables");
 
-    let time_limit =
-        pbs::get_remaining_timelimit(&manager_job_id).expect("Could not get PBS timelimit");
+    let time_limit = match pbs::get_remaining_timelimit(&manager_job_id) {
+        Ok(time_limit) => Some(time_limit),
+        Err(error) => {
```

Comment:
``or_else``

Reply:
We need to convert `Result` to `Option`, so it's not that easy here. I would like to use `inspect`, but it's unstable ATM.

Reply:
Ah, sorry I missed this.